Wazi_Deploy/Reporting-pipeline/README.md

```diff
@@ -0,0 +1,76 @@
+# Gitlab IBM Wazi Deploy query template
+
+This template provides a [.gitlab-ci.yml](.gitlab-ci.yml) definition file to setup a pipeline for query the deployment information from the Wazi Deploy evidence file and create a deployed artifacts report.
+
+## Overview and capabilities
```

Comment:
Can you please split the description into 

* `queryTemplate.yml` and `renderer.yml` (with brief links to the Wazi Deploy documentation on reporting). 
* Pipeline Templates to trigger the wazideploy-evidence commands.

The gitlab description should then be part of a README.md within the Gitlab subdirectory.